MapboxNavigation/PollyVoiceController.swift

```diff
@@ -141,10 +141,15 @@ public class PollyVoiceController: RouteVoiceController {
     }
     
     public override func speak(_ instruction: SpokenInstruction) {
+        guard let routeProgress = routeProgress else {
+            assert(false, "`routeProgress should not be nil`")
```

Comment:
assert(routeProgress != nil, "routeProgress should not be nil.")

Reply:
I mean, then we have to `!` all instances of routeProgress below